performance.rst

```diff
@@ -125,82 +124,6 @@ your deploy process:
   Prevents Composer from scanning the file system for classes that are not
   found in the class map.
 
-Caching the Autoloader with APC
--------------------------------
-
-Another solution is to cache the location of each class after it's located
-the first time. Symfony comes with a class - :class:`Symfony\\Component\\ClassLoader\\ApcClassLoader` -
-that does exactly this. To use it, just adapt your front controller file.
-If you're using the Standard Distribution, make the following changes::
-
-    // app.php
-    // ...
-    
-    use Symfony\Component\ClassLoader\ApcClassLoader;
-
-    // do not use $loader as a variable name here as it would
-    // be overwritten when loading the bootstrap.php.cache file
-    $classLoader = require __DIR__.'/../app/autoload.php';
-    include_once __DIR__.'/../app/bootstrap.php.cache';
-
-    // Use APC for autoloading to improve performance
-    // Change 'sf2' by the prefix you want in order
-    // to prevent key conflict with another application
-    $loader = new ApcClassLoader('sf2', $classLoader);
-    $loader->register(true);
-
-    // ...
-
-For more details, see :doc:`/components/class_loader/cache_class_loader`.
-
-.. note::
-
-    When using the APC autoloader, if you add new classes, they will be found
-    automatically and everything will work the same as before (i.e. no
-    reason to "clear" the cache). However, if you change the location of a
-    particular namespace or prefix, you'll need to flush your APC cache. Otherwise,
-    the autoloader will still be looking at the old location for all classes
-    inside that namespace.
-
-.. index::
-   single: Performance; Bootstrap files
-
-Use Bootstrap Files
```

Comment:
not recommended anymore, esp. with php 7

Reply:
technically this is still relevant for php 5 (as you know), but I'm +1 for removing it... because users already have this code - PHP 5 users don't need to take any action.